libraries/AP_Motors/AP_Motors6DOF.h

```diff
@@ -41,6 +41,8 @@ class AP_Motors6DOF : public AP_MotorsMatrix {
     // output_to_motors - sends minimum values out to the motors
     void output_to_motors() override;
 
+    float get_throttle_in_bidirectional() const { return constrain_float(2*(_throttle_in - 0.5f), -1.0f, 1.0f); }
```

Comment:
make a note why we are doing this (the controllers are in charge of the throttle input, so this gives vehicle access/visibility to the output of those controllers)

Reply:
Done